<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathPropertiesDialog</class>
 <widget class="QDialog" name="PathPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path Properties</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/qtgameengine/icons/path-file.png</normaloff>:/qtgameengine/icons/path-file.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolBar">
        <property name="windowTitle">
         <string>toolBar</string>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <addaction name="actionOk"/>
        <addaction name="separator"/>
        <addaction name="actionUndo"/>
        <addaction name="separator"/>
        <addaction name="actionClear"/>
        <addaction name="actionReverse"/>
        <addaction name="actionShift"/>
        <addaction name="actionMirrorHorizontally"/>
        <addaction name="actionFlipVertically"/>
        <addaction name="actionRotate"/>
        <addaction name="actionScale"/>
        <addaction name="separator"/>
        <addaction name="actionShiftLeft"/>
        <addaction name="actionShiftRight"/>
        <addaction name="actionShiftTop"/>
        <addaction name="actionShift_bottom"/>
        <addaction name="actionCenterView"/>
        <addaction name="separator"/>
        <addaction name="separator"/>
        <addaction name="actionGrid"/>
        <addaction name="actionRoom"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>&amp;Name:</string>
           </property>
           <property name="buddy">
            <cstring>lineEditName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditName">
           <property name="toolTip">
            <string>The name of the path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelX">
             <property name="text">
              <string>&amp;X:</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxX</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBoxX">
             <property name="toolTip">
              <string>X-coordinate of the point</string>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelY">
             <property name="text">
              <string>&amp;Y:</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxY</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBoxY">
             <property name="toolTip">
              <string>Y-coordinate of the point</string>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelSp">
             <property name="text">
              <string>s&amp;p:</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxSp</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBoxSp">
             <property name="toolTip">
              <string>Relative speed at this point (100 = default)</string>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="toolTip">
              <string>Add the point to the path</string>
             </property>
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInsert">
             <property name="toolTip">
              <string>Insert the point before the current one</string>
             </property>
             <property name="text">
              <string>&amp;Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDelete">
             <property name="toolTip">
              <string>Delete the point from the path</string>
             </property>
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBoxConnectionKind">
             <property name="toolTip">
              <string>Indicate the type of connections between the points</string>
             </property>
             <property name="title">
              <string>connection kind</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioButtonStraight">
                <property name="text">
                 <string>&amp;Straight lines</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSmooth">
                <property name="text">
                 <string>S&amp;mooth curves</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labelPrecision">
               <property name="text">
                <string>P&amp;recision:</string>
               </property>
               <property name="buddy">
                <cstring>spinBoxPrecision</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBoxPrecision">
               <property name="toolTip">
                <string>Precision with which to create smooth curves</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBoxClosed">
             <property name="text">
              <string>Closed</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,0">
       <item>
        <widget class="PathPointsWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStatusBar" name="statusbar"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOk">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/ok.png</normaloff>:/qtgameengine/icons/ok.png</iconset>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="toolTip">
    <string>Close the form, saving the changes</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/undo.png</normaloff>:/qtgameengine/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the last changes in the path</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/new.png</normaloff>:/qtgameengine/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear the path</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/rotate.png</normaloff>:/qtgameengine/icons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
   <property name="toolTip">
    <string>Reverse the path</string>
   </property>
  </action>
  <action name="actionShift">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/move.png</normaloff>:/qtgameengine/icons/move.png</iconset>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
   <property name="toolTip">
    <string>Shift the path</string>
   </property>
  </action>
  <action name="actionMirrorHorizontally">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/flip-horizontal.png</normaloff>:/qtgameengine/icons/flip-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Mirror horizontally</string>
   </property>
   <property name="toolTip">
    <string>Mirror the path horizontally</string>
   </property>
  </action>
  <action name="actionFlipVertically">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/flip-vertical.png</normaloff>:/qtgameengine/icons/flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip vertically</string>
   </property>
   <property name="toolTip">
    <string>Flip the path vertically</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/rotate.png</normaloff>:/qtgameengine/icons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate the path</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/scale.png</normaloff>:/qtgameengine/icons/scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Scale the path</string>
   </property>
  </action>
  <action name="actionShiftLeft">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/arrow-left.png</normaloff>:/qtgameengine/icons/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Shift Left</string>
   </property>
   <property name="toolTip">
    <string>Shift the view to the left</string>
   </property>
  </action>
  <action name="actionShiftRight">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/arrow-right.png</normaloff>:/qtgameengine/icons/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Shift right</string>
   </property>
   <property name="toolTip">
    <string>Shift the view to the right</string>
   </property>
  </action>
  <action name="actionShiftTop">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/arrow-up.png</normaloff>:/qtgameengine/icons/arrow-up.png</iconset>
   </property>
   <property name="text">
    <string>Shift top</string>
   </property>
   <property name="toolTip">
    <string>Shift the view to the top</string>
   </property>
  </action>
  <action name="actionShift_bottom">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/arrow-down.png</normaloff>:/qtgameengine/icons/arrow-down.png</iconset>
   </property>
   <property name="text">
    <string>Shift bottom</string>
   </property>
   <property name="toolTip">
    <string>Shift the view to the bottom</string>
   </property>
  </action>
  <action name="actionCenterView">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/center.png</normaloff>:/qtgameengine/icons/center.png</iconset>
   </property>
   <property name="text">
    <string>Center view</string>
   </property>
   <property name="toolTip">
    <string>Center view around the path</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/grid.png</normaloff>:/qtgameengine/icons/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle the showing of the grid</string>
   </property>
  </action>
  <action name="actionRoom">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/qtgameengine/icons/room.png</normaloff>:/qtgameengine/icons/room.png</iconset>
   </property>
   <property name="text">
    <string>Room</string>
   </property>
   <property name="toolTip">
    <string>Indicate the room to show as background</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathPointsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/pathpointswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOk</sender>
   <signal>triggered()</signal>
   <receiver>PathPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
